.gitignore

```diff
@@ -91,3 +91,9 @@ gradle-app.setting
 */Data/
 # End of https://www.gitignore.io/api/gradle,intellij
 Logs/
+
+# VSCode Adds
```

Comment:
Should we add .vscode also? Or is there helpful data in there that should be checked into source control?

Reply:
I didn't want to pollute with the VS Code stuff. But I can see if there is anything of use in them.

Reply:
Probably best to just exclude `.vscode` here to keep IDE-specific stuff out as much as possible